### PR TITLE
Add docs for ordering

### PR DESCRIPTION
**Pull request by violinyanev**
Based on Achim's demo and docs.

### CODE REVIEW

File: ordering/manual.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+In this chapter we demonstrate different ordering techniques and how to use and combine them.
+
+# Contents of a Ramses Composer project
+
+In order to understand how ordering of content works, it's essential to understand
+the contents of a Ramses Composer project first.
+
+A newly created project contains following default objects (which can be deleted later):
+
+* A scene graph with one `Node`, one (child) `MeshNode`, and a camera
```
Comment (bojackHaasman) [inline ordering/manual.md:35]:
```suggestion
* A scene graph with one `Node`, one (child) `MeshNode`, and a `Camera`
```

File: ordering/manual.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+The way this works is:
+
+* The `RenderLayer` collects all nodes from the scene with the given tag (`render_main`) and their child nodes recursively
+* The `RenderPass` renders everything from the `RenderLayer` using the selected `Camera`
+* The order of the meshes is influenced by multiple factors, as we will learn in the next sections
+
+# Ordering techniques
+
+The order of rendering is influenced by three factors and can be conbined in different ways.
```
Comment (bojackHaasman) [inline ordering/manual.md:63]:
```suggestion
The order of rendering is influenced by three factors and can be combined in different ways.
```

File: ordering/manual.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+![](./docs/color_tags_1.png)
+
+Notice that each colored MeshNode has its own tag, and all tags (red, green, blue) are listed in the
+`RenderLayer` renderable tags property field. If you edit the field, you will see that each of the tags
+has its own order number assigned to it:
+
+![](./docs/color_tags_2.png)
+
+Furthermore, the `Render order` property further down is set to `Render order value in Renderable tags`.
+This means that this `RenderLayer` will sort its content based on the numbers assigned with each tag.
```
Comment (bojackHaasman) [inline ordering/manual.md:98]:
```suggestion
This means that this `RenderLayer` will sort its content based on the numbers assigned to each tag.
```

File: ordering/manual.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+In particular, it will sort tags based on their Render Order (starting with lowest), then sort all
+mesh nodes which fit to one or more of the tags, and will sort them based on their tags order.
```
Comment (bojackHaasman) [inline ordering/manual.md:100]: This sentence is quite hard to understand. Maybe explain with one simple example with multiple tags for one meshNode how the sorting works

File: ordering/manual.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+Notice that we still need separate `RenderLayers` too. RenderPasses don't group or filter
+objects in any way - that's done by the `RenderLayers` and their tags. `RenderPasses` can
+only reference `RenderLayers` which have already collected and possibly sorted their content.
+
+In this setup, each `RenderPass` has a single `RenderLayer` attached to it, instead of all three.
+The render order of the `RenderPasses` is defined by their `Order` property:
+
+![](./docs/renderpass_order.png)
+
+Countrary to `RenderLayer` tags, `RenderPasses` must be ordered with unique order. If you have two
```
Comment (bojackHaasman) [inline ordering/manual.md:144]:
```suggestion
Contrary to `RenderLayer` tags, `RenderPasses` must be ordered with unique order. If you have two
```

**Review by bojackHaasman — CHANGES_REQUESTED** [review]

File: ordering/manual.md (hunk 4 above)
Comment (violinyanev) [reply ordering/manual.md]: Good suggestion - can you please recheck?

**Review by bojackHaasman — APPROVED** [review]

**Pull request merged**